py3dtilers/CityTiler/__init__.py

```diff
@@ -1,4 +1,14 @@
+from .citym_cityobject import CityMCityObjects
+from .citym_building import CityMBuildings
+from .citym_relief import CityMReliefs
+from .citym_waterbody import CityMWaterBodies
+from .citym_bridge import CityMBridges
 from .CityTiler import main
+from .CityTiler import Tiler
```

Comment:
`Tiler` is a class from the Common module and is [already exposed](https://github.com/VCityTeam/py3dtilers/blob/master/py3dtilers/Common/__init__.py#L14).

Maybe this line should expose `CityTiler` ?

Reply:
You're perfectly right. My bad. Fixed.